warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -535,6 +537,35 @@ models:
       - name: _is_current
       - name: _valid_from
       - name: _valid_to
+  - name: dim_mobility_mart_providers
+    description: |-
+      Listing of all Transit Agencies (TAs) in California with respective data fields needed for the Mobility Marketplace provider map.
+    columns:
+      - name: agency_name
+        description: The name of the TA as specified in the National Transit Database (NTD).
+      - *org_ntd_id
```

Comment:
I'm wondering if you might want unique and not null tests on this column, since my understanding is that you want one row per NTD ID